xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -386,7 +386,7 @@ bool CDVDDemuxFFmpeg::Open(CDVDInputStream* pInput, bool streaminfo, bool filein
 
     m_pFormatContext->pb = m_ioContext;
 
-    AVDictionary *options = NULL;
+    AVDictionary *options = GetFFMpegOptionsFromInput();
```

Comment:
@FernetMenta , have you read the log in the post http://forum.kodi.tv/showthread.php?tid=312616.
I have same problem, if ffmpeg options are not for custom io handling can you tell me how to pass an User-Agent or a cookie ?
Options are perhaps not for them, but needed.